src/common/KernelBase.hpp

```diff
@@ -570,6 +570,7 @@ class KernelBase
   cali_id_t Bytes_Rep_attr;
   cali_id_t Flops_Rep_attr;
   cali_id_t BlockSize_attr;
+  std::map<std::string, cali_id_t> Features;
```

Comment:
How about adding `_attrs` to this variable name?

Reply:
Done [07baca4](https://github.com/LLNL/RAJAPerf/pull/456/commits/07baca41feeaf4040b9c047e1523d02ac075d1cb)